spec/arproxy_spec.rb

```diff
@@ -24,128 +24,189 @@ def execute(sql, name)
   module ::ActiveRecord
     module ConnectionAdapters
       class DummyAdapter
-        def execute(sql, name = nil)
-          {:sql => sql, :name => name}
+        ADAPTER_NAME = 'Dummy'
+
+        def execute1(sql, name = nil, **kwargs)
+          { sql: sql, name: name, kwargs: kwargs }
+        end
+
+        def execute2(sql, name = nil, binds = [], **kwargs)
+          { sql: sql, name: name, binds: binds, kwargs: kwargs }
         end
       end
+      Arproxy::ConnectionAdapterPatch.register_patches('Dummy', patches: [:execute1], binds_patches: [:execute2])
     end
   end
 
   let(:connection) { ::ActiveRecord::ConnectionAdapters::DummyAdapter.new }
-  subject { connection.execute "SQL", "NAME" }
   after(:each) do
     Arproxy.disable!
+    Arproxy.clear_configuration
```

Comment:
Q. Is this necessary even if we run `Arproxy.clear_configuration` on each `before` block?

Reply:
It was indeed redundant, so deleted it 👍 
https://github.com/cookpad/arproxy/pull/32/commits/74de4a4552412caafd92411e5d5e3d7e4679900c